lib/yao/resources/floating_ip.rb

```diff
@@ -18,5 +18,48 @@ class FloatingIP < Base
     def router
       @router ||= Yao::Router.get(router_id)
     end
+
+    # @param [Yao::Resources::Port]
+    # @return [Yao::Resources::FloatingIP]
+    def associate_port(port)
+      self.class.associate_port(id, port.id)
+    end
+
+    # @return [Yao::Resources::FloatingIP]
+    def disassociate_port
+      self.class.disassociate_port(id)
+    end
+
+    class << self
+
+      # @param id [String] ID of floating_ip
+      # @param port_id [String] ID of port
+      # @return [Yao::Resources::FloatingIP]
+      def associate_port(id, port_id)
+
+        param = {
+          floatingip: {
+            port_id: port_id
+          }
+        }
+
+        res = PUT([resources_path, id ].join('/'), param.to_json)
+        resource_from_json(res.body)
```

Comment:
Yao::Resource::Base.updateを使うとよさそうです！
https://github.com/yaocloud/yao/blob/master/lib/yao/resources/restfully_accessible.rb#L174

```suggestion
        update(id, port_id: port_id)
```

Reply:
便利!